compiler/gradient_ops.cc

```diff
@@ -380,7 +380,7 @@ void ReduceMeanGradFn(GradientOpContext* gc) {
     // TODO(hamaji): Need some check for `axes` and `keepdims`.
     Value* gy = gc->gy(0);
     Value* shape = gb.Op(Node::kShape, {gc->x(0)});
-    Value* zero = gb.Const(Type(Dtype::kInt64, {}), {0});
+    Value* zero = gb.ScalarConst<int64_t>(0, Dtype::kInt64);
```

Comment:
Do we still need the explicit template parameter `<int64_t>`? Here and elsewhere.